examples/hana-scale-up-perf-optimized-azure.yaml

```diff
@@ -304,18 +304,23 @@ groups:
     description: "Check OS and package versions are supported and up-to-date"
     checks:
       - id: 2.2.1
-        description: "Check the base product"
+        description: "Check SLES4SAP is installed"
         # Assume the perf-optimized scenario and
         # check only that the base probuct is always sles_sap.prod (case-insensitively)
         audit: 'readlink /etc/products.d/baseproduct | sed -e "s/\(.*\)/\L\1/"'
         tests:
           test_items:
             - flag: 'sles_sap.prod'
         remediation: |
-          Install a SUSE supported OS release
+          ## Abstract
+          You need to use SUSE Linux Enterprise Server for SAP Applications
+          ## Remediation
+          SAPHanaSR is only supported on SLES4SAP , please install SLES4SAP.
```

Comment:
Also, we should never use the `SLES4SAP` moniker in user-facing stuff, we should always use the full product name, `SUSE Linux Enterprise Server for SAP Applications`.